hello-helm/k8s/values.yaml

```diff
@@ -0,0 +1,9 @@
+environment: demo
```

Comment:
The name of the app is `demo` (at least, it seems to have emerged that way)—not sure what the environment should be. Maybe it should be `prod` to start with (or equivalent) and then we introduce the idea of staging environments?

Reply:
Should we rename the repo to demo so it matches the app name?